election-portal/app/election-breakdown/modules/stateMap.tsx

```diff
@@ -130,92 +127,136 @@ const StateMap: React.FC<RTCMapProps> = ({ raceType, year, stateName }) => {
+          },
       },
       mapNavigation: {
-        enabled: false,
-        enableButtons: false,
+          enabled: true,
+          enableMouseWheelZoom: true,
+          enableButtons: false,
       },
       colorAxis: colorAxis,
-      tooltip: {
-        formatter: function (this: any) {
-          return (
-            "<b>" +
-            this.point.properties.NAME + " County" +
-            "</b>"
-          );
-        },
-        style: {
-          fontFamily: "gelica, book antiqua, georgia, times new roman, serif",
-        },
-      },
       legend: {
-        itemStyle: {
-          fontFamily: "gelica, book antiqua, georgia, times new roman, serif",
-        },
+          itemStyle: {
+              fontFamily: "gelica, book antiqua, georgia, times new roman, serif",
+          },
       },
       series: [
-        {
-          showInLegend: false,
-          type: "map",
-          mapData: mapData,
-          data: presData,
-          joinBy: 'GEOID',
-          nullColor: "#FFFFFF",
-          name: "Predicted Margin",
-          borderColor: "black",
-          borderWidth: 1,
-          events: {
-            click: function(event: any) {
-              const countyName = event.point.properties.NAME;
-              alert(`You just clicked ${countyName}`);
-            },
+          {
+              showInLegend: false,
+              type: "map",
+              mapData: mapData,
+              data: presData,
+              joinBy: "GEOID",
+              nullColor: "#FFFFFF",
+              name: "Counties",
+              borderColor: "black",
+              borderWidth: 2,
+              states: {
+                  hover: {
+                      borderColor: 'lightgreen',
+                  }
+              },
+              dataLabels: {
+                  format: "{point.properties.NAME}",
+                  style: {
+                      fontFamily: "gelica, book antiqua, georgia, times new roman, serif",
+                  },
+                  padding: 10,
+              },
+              tooltip: {
+                  pointFormat: "{point.properties.NAME} County",
+              },
           },
-          dataLabels: {
-            // enabled: true,
-            format: "{point.properties.NAME}",
-            style: {
-              fontFamily:
-                "gelica, book antiqua, georgia, times new roman, serif",
-            },
+          {
+              // Series for cities (mappoint)
+              type: 'mappoint',
+              name: 'Cities',
+              color: '#D9D9D9', 
+              data: cityData,
+              dataLabels: {
+                  enabled: true,
+                  allowOverlap: true,  
+                  style: {
+                      fontSize: '12px', 
+                      fontWeight: 'bold',
+                      textOutline: '1px contrast',
+                      color: '#FFFFFF',
+                      fontFamily: 'Gelica Arial, sans-serif',
```

Comment:
copy the font stack from the brand guide's last page or look elsewhere in our code base to find it

Reply:
oh wait you have the right font stack in this file a few lines above. that is what you should use